benchmark/aabench/int.d

```diff
@@ -0,0 +1,20 @@
+/**
+ * Benchmark int hashing.
+ *
+ * Copyright: Copyright Martin Nowak 2011 - 2015.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:    Martin Nowak
+ */
+import std.random;
+
+void main(string[] args)
+{
+    auto rnd = Xorshift32(33);
+
+    int[int] aa;
+    foreach (_; 0 .. 10)
+        foreach (__; 0 .. 100_000)
+            ++aa[uniform(0, 100_000, rnd)];
+
+    assert(aa.length == 99994);
```

Comment:
You should also use `if(aa.length != 99994) assert(0);` here as the benchmrks are usually not build in debug mode. Using `enforce` instead of `assert`avoids the boilerplate.